libraries/AC_Avoidance/AP_OABendyRuler.cpp

```diff
@@ -649,6 +655,9 @@ bool AP_OABendyRuler::calc_margin_from_object_database(const Location &start, co
     if (!start.get_vector_from_origin_NEU(start_NEU) || !end.get_vector_from_origin_NEU(end_NEU)) {
         return false;
     }
+    if (start_NEU == end_NEU) {
```

Comment:
I have inlcluded this bit as an extra safety check. I found this as a potential bug. Its not really needed in this PR.
What happens is, if the two points are the same, `Vector3f::closest_distance_between_line_and_point` returns 0
therefore the line
`Vector3f::closest_distance_between_line_and_point(start_NEU, end_NEU, point_cm) * 0.01f - item.radius;` would falsely return a negative margin.. Something that can be scary mid flight..